projects/PM_hash_navigation/main.cc

```diff
@@ -1,27 +1,29 @@
 //main
+#include "pm_control.h"
+#include "pm_hasher.h"
+#include "snapshot_db.h"
 
 // BoB robotics includes
-#include "common/logging.h"
+// #include "common/logging.h" // not needed in new BoB robotics
+#include "common/main.h"
```

Comment:
These changes, like a few of the other ones, might not work with old BoB robotics